gradle/libs.versions.toml

```diff
@@ -0,0 +1,86 @@
+[versions]
+
+## compilation ##
+jvm = "11"
+kotlin = "1.8.10"
+kotlin-languageLevel = "1.4"
+
+
+## libs ##
+kotlin-plugin = "213-1.8.10-release-430-IJ6777.52"
+kotlinx-coroutines = "1.6.3"
+kotlinx-html = "0.7.5"
+jsoup = "1.15.3"
```

Comment:
We can inline these once-use versions.

Reply:
I think it's best to define all the versions at the top, because it's consistent, it's easier to scan the top to see the versions, and if any versions do need to be re-used then then it's less work (personally I find editing `libs.versions.toml` a bit fiddly, so I try to avoid it).

Reply:
We can easy to know which versions are reused after inlining unnecessary ones, either way.

Reply:
I think it should be consistent - either all versions are at the top, or all of the versions are inlined, otherwise it might eventually lead to chaos. At the moment, I personally don't have any preference, so I'm fine with either approach :+1: